docs/conf.py

```diff
@@ -19,6 +19,7 @@
 #
 import os
 import sys
+import sparse
```

Comment:
This should be moved below the `sys.path.insert`, otherwise it won't work. RTD runs Sphinx from the `docs` directory.